### PR TITLE
integrated financials backend API into explore tab

### PR DESCRIPTION
**Pull request by StefanPitur**

### CODE REVIEW

**Comment by mileswatson** [comment]
@StefanPitur Will hold off approving until Kate's PR #101 is merged (or not, if you decide against it).

**Review by katelkq — APPROVED** [review]

**Review by mileswatson — APPROVED** [review]

**Comment by StefanPitur** [comment]
@mileswatson @katelkq ready to merge?

**Comment by mileswatson** [comment]
@StefanPitur yep :)

**Pull request merged**
